docs-chef-io/content/workstation/getting_started.md

```diff
@@ -19,60 +19,37 @@ new Chef Workstation installation. If you do not have Chef Workstation
 installed, see its [installation guide](/workstation/install_workstation/)
 before proceeding further.
 
-## Configure Ruby Environment
+## Install a Code Editor
 
-For many users of Chef, the version of Ruby that is included in Chef
-Workstation should be configured as the default version of Ruby on your
-system.
-
-{{< note >}}
-
-These instructions are intended for macOS and Linux users.
-
-{{< /note >}}
-
-1. Open a terminal and enter the following:
-
-    ``` bash
-    which ruby
-    ```
-
-    which will return something like `/usr/bin/ruby`.
-
-2. To use Chef Workstation-provided Ruby as the default Ruby on your system, edit the `$PATH` and `GEM` environment variables to include paths to Chef Workstation. For example, on a machine that runs Bash, run:
-
-    ``` bash
-    echo 'eval "$(chef shell-init bash)"' >> ~/.bash_profile
-    ```
+A good visual code editor is not a requirement for working with Chef
+Infra, but a good code editor can save you time. A code editor should
+support the following: themes, plugins, snippets, syntax Ruby code
+coloring/highlighting, multiple cursors, a tree view of the entire
+folder/repository you are working with, and a Git integration.
 
-    where `bash` and `~/.bash_profile` represents the name of the shell.
+These are a few common editors:
 
-    If zsh is your preferred shell then run the following:
+- [Visual Studio Code (free/open source)](http://code.visualstudio.com)
+- [GitHub Atom - (free/open source)](http://atom.io)
 
-    ``` bash
-    echo 'eval "$(chef shell-init zsh)"' >> ~/.zshrc
-    ```
+Chef Infra support in editors:
 
-3. Run `which ruby` again. It should return
-    `/opt/chef-workstation/embedded/bin/ruby`.
+- [VSCode Chef Infra Extension](https://marketplace.visualstudio.com/items?itemName=chef-software.Chef)
+- [Chef on Atom](https://atom.io/packages/language-chef)
+## Set up Ruby
 
-{{< note >}}
+While you don't have to know the Ruby programming language to use Chef, you do need to make sure that your computer knows where to find Ruby in order to run.
 
-Using Chef Workstation-provided Ruby as your system Ruby is optional.
-For many users, Ruby is primarily used for authoring Chef cookbooks. If
-that's true for you, then using the Chef Workstation-provided Ruby is
-recommended.
+* If you don't plan on writing a lot of Chef cookbooks and you don't use Ruby for other purposes, then set up Ruby with an [environment variable]({{< ref "#Add Ruby as an Environment Variable" >}})
```

Comment:
I don't think this is entirely correct. I like the structure and tone, but we need to get the reasons right:

- You just want chef to work and your not a ruby developer elsewhere: use our ruby
- You do ruby development with your system ruby: env vars
- You use RVM: let's not document that. It's all a mess of terribleness